RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -162,6 +163,12 @@ - (bool) isTablet
   return [[UIDevice currentDevice] userInterfaceIdiom] == UIUserInterfaceIdiomPad;
 }
 
+- (bool) isPinOrFingerprintSet
+{
+  LAContext *context = [[LAContext alloc] init];
+  return ([context canEvaluatePolicy:LAPolicyDeviceOwnerAuthenticationWithBiometrics error:nil]);
```

Comment:
Good catch.  LAPolicyDeviceOwnerAuthentication will behave as intended, but only for iOS 9.0+.  In the case of iOS <9.0, I can fall back on LAPolicyDeviceOwnerAuthenticationWithBiometrics and write a note about this in the README.